src/LiveComponent/assets/dist/set_deep_data.js

```diff
@@ -0,0 +1,80 @@
+"use strict";
+
+Object.defineProperty(exports, "__esModule", {
```

Comment:
This is definitely one of the things that needs to be done before release - I was thinking we should use bundler to properly put this together. Thanks for mentioning this :)

Reply:
side note: should we even transpile those files for IE11 compat ? It increases the size for all projects even if they don't need IE11 support. It might be better to let projects needing IE11 to include those packages in the transpiled ones in the webpack config.

Reply:
I was actually thinking about exactly this too. Either not transpiling for IE11 and either document that users should include a polyfill package we provide (that's why I have the starting polyfill as a separate file right now) or just polyfill themselves. There is some work to be done here to optimize but make IE11 support clear